pkg/server/admin.go

```diff
@@ -897,6 +897,38 @@ func (s *adminServer) tableDetailsHelper(
 		resp.CreateTableStatement = createStmt
 	}
 
+	// MVCC Garbage result.
+	row, cols, err := s.server.sqlServer.internalExecutor.QueryRowExWithCols(
+		ctx, "admin-show-mvcc-garbage-pct", nil,
+		sessiondata.InternalExecutorOverride{User: userName},
+		`WITH 
+			range_stats AS (
+				SELECT 
+					crdb_internal.range_stats(start_key) AS d 
+				FROM crdb_internal.ranges_no_leases 
+				WHERE table_id = '$1'::REGCLASS), 
```

Comment:
I don't think you need to quote this. In fact, I'm surprised that this works with quotes around it like this.

Reply:
Fixed